test_get_x_posts.py

```
@@ -0,0 +1,229 @@
+    """patch済みsubprocess.runに渡されたコマンド引数リストの一覧。"""
+    return [call.args[0] for call in mock_run.call_args_list]
+
+
+@pytest.fixture
+def pipeline_env(tmp_path, monkeypatch):
+    """main()を外部依存なしで実行するための最小環境。"""
+    monkeypatch.setattr(x2b, "STATE_FILE", tmp_path / "seen.json")
+    monkeypatch.setattr(x2b, "USERS_FILE", tmp_path / "users.json")
+    monkeypatch.setattr(x2b, "LOCK_FILE", tmp_path / ".x2b.lock")
+
+    # キャッシュヒットさせ、twitter-cliの起動（user取得）を防ぐ
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)